### ISSUE TITLE
Fix failing pa11y tests

### ISSUE THREAD

**Issue by timgentry** [HealthDataInsight/design_system#274]
The accessibility github workflows are failing and need fixing. We need to:

* Expand the number of pages tested, now that we use the dummy app as a design system demonstration site
* Fix the underlying issue(s) in the design system (e.g. missing aria tags)
* Fix any issues in the use of the design system in those page (e.g. using the same HTML ID more than once on a page)
